packages/neutrino/bin/start.js

```diff
@@ -43,7 +43,7 @@ module.exports = (middleware, args, cli) => {
         });
       } else {
         const { devServer } = compiler.options;
-        const url = `${devServer.https ? 'https' : 'http'}://${devServer.public}`;
+        const url = `${devServer.https ? 'https' : 'http'}://${devServer.public}:${devServer.port}`;
```

Comment:
This was my concern as well, because we used to have this. `public` should also have the port built into it, since the value for `public` can be host:port. We need to normalize the value in dev-server differently to account for this.

Reply:
I think I know what you mean. Deleting the last commit and pushing something else now.